Makefile

```diff
@@ -3,6 +3,7 @@ GOTOOLS := \
 	github.com/git-chglog/git-chglog/cmd/git-chglog \
 	github.com/golang/dep/cmd/dep \
 	golang.org/x/tools/cmd/cover \
+	github.com/stretchr/testify/assert \
```

Comment:
Testify shouldn't be globally installed but rather installed and  tracked through `dep`. You can run `dep ensure` to add it to the Gopkg.lock file instead of adding it as a global dependency.